src/gui/widgets/text_box.cpp

```diff
@@ -114,6 +114,8 @@ text_box::text_box()
 			&text_box::signal_handler_left_button_up, this, _2, _3));
 	connect_signal<event::LEFT_BUTTON_DOUBLE_CLICK>(std::bind(
 			&text_box::signal_handler_left_button_double_click, this, _2, _3));
+
+	update_offsets();
```

Comment:
Should be. Those two functions update the `pango_text` settings, while `update_offsets` handles widget parameters.